docs/program-setup/bare_metal/c_small_libs.md

```diff
@@ -58,5 +63,5 @@ After you have completed the steps above, add `small` to the `supported_c_libs`
 ```
 
 ### Note on uARM toolchain
-
+<!--I think all refrences to uArm should be removed at this point - including the scatter file bits above-->
```

Comment:
We need to keep the scatter file information somewhere a it is very useful.

Reply:
I'm just worried about the uArm bit, since we deprecated it.

Reply:
But we have not removed all the TOOLCHAIN_ARM_MICRO folders in the code base, specifically for this reason.

Reply:
Oh, I didn't know that. So this page is still 100% accurate?